src/main/java/no/ntnu/mycbr/rest/ValueRange.java

```diff
@@ -13,6 +13,7 @@
  */
 public class ValueRange {
 
+    private static final String ATTRIBUTE_TYPE = "attribute_type"; 
     private HashMap<String, Object> attributes = new HashMap<String, Object>();
```

Comment:
What is `attributes` map for? Is *ATTRIBUTE_TYPE* in line with other values?
javadoc required.

Reply:
Added comments in commit: e182cfa